rskj-core/src/main/java/co/rsk/config/RemascConfigFactory.java

```diff
@@ -34,24 +42,57 @@ public class RemascConfigFactory {
     private static final Logger logger = LoggerFactory.getLogger("RemascConfigFactory");
 
     private ObjectMapper mapper;
-    private String configPath;
+    private final Map<String, RemascConfig> remascConfigMap = new ConcurrentHashMap<>();
 
     public RemascConfigFactory(String remascConfigFile) {
         this.mapper = new ObjectMapper();
-        this.configPath = remascConfigFile;
+        logger.info("RemascConfigFactory remascConfigFile = {} ", remascConfigFile);
+
+        try (InputStream is = RemascConfigFactory.class.getClassLoader().getResourceAsStream(remascConfigFile)) {
+            String sis = new BufferedReader(new InputStreamReader(is, StandardCharsets.UTF_8))
+                    .lines()
+                    .collect(Collectors.joining("\n"));
+
+            logger.info("RemascConfigFactory sis = {} ", sis);
```

Comment:
Do we need this new precise logging? Before we were logging only one error.

Reply:
I want to log as much as possible to track the scenario where the UnzipException occurred. If it does not happen for some time, we can remove these logs and assume this refactor fixes it, but yeah we can just remove it if you think this will not happen again after this change.